docs/src/main/sphinx/installation/cli.rst

```diff
@@ -37,6 +37,27 @@ password value to avoid the prompt.
 Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
+External authentication - SSO
+-----------------------------
+
+The  ``--external-authentication`` option is used for browser-based SSO
+authentication, as detailed in :doc:`/security/oauth2`. With this configuration,
+the CLI displays an URI that you have to open in a web browser for authentication.
+Due to the above, this can only be used when you have an access to a web-browser,
+so that you can copy-paste the displayed uri.
+
+The expected behaviour is as follows:
+ - user opens up CLI with ``--external-authentication`` option enabled,
+ - CLI opens up and allows a user to start writing queries,
+ - on a first query, a request to open up a browser will be displayed,
+ - after completing the login, user's query will be executed and its results displayed,
+ - every other query will not require user to login again, as long as the token is valid,
+ - if coordinator decides to reject the token (for example due to its timeout), a user
+   will be asked to open a browser again and go through the flow there.
+
+Note that in order to use external authentication, tls needs to be enabled as well.
+For more details take a look at :doc:`/security/cli`
```

Comment:
Remove this link since it is not valid for TLS with the cli ..

Reply:
Ok I've pointed it there as there is a message what to do with it for Kerberos, but you are right that security/tls is even better.